packages/video_player/video_player_web/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 ## NEXT
 
+* Adds caching functionality for network video.
```

Comment:
No need to modify anything in `video_player_web` for this feature, right? Unless you're going to implement the new methods that were added in the platform interface as NOOPs so they don't crash.

If this is only going to be metadata changes for the web, you should be able to revert them!

Reply:
You are correct, it seems that, and I have tested this on different browsers, caching is by default supported (probably based on browser settings off course. This is also what I was expecting. Therefore we don't have to do anything yet for web. I'll cleanup some web changes.

Reply:
Yes, caching is on by default in all web browsers. Users can tweak its behavior a little bit. Server response headers can be used to control its behavior (like [`cache-control: no-cache`](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#no-cache)). In general it is *hard* to change the default (in the browser, you'd need a special service worker to override the default behavior). Thanks for cleaning this up!